### PR TITLE
Fixes #1132: Send username of the user in /cms/getSkillFeedback.json

### LINKED ISSUE #1132: Send user name if exists in the getSkillFeedback response
**Actual Behaviour**
* Not implemented yet

**Expected Behaviour**
* Send user name if exists in the getSkillFeedback response

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by akshatnitd**
Fixes #1132 

Changes: [Add here what changes were made in this issue and if possible provide links.]
* Send username of the user in /cms/getSkillFeedback.json

### CODE REVIEW

**Comment by arundhati24** [comment]
@akshatnitd In case the user has not updated her/his username, in that case an empty string is being returned. So, in the client, for such users should the anonymised email be displayed?

File: src/ai/susi/server/api/cms/GetSkillFeedbackService.java [hunk 1]
```diff
@@ -135,4 +136,19 @@ public static String getUserAvatar(String email) {
...
         return avatar_url;
     }
+
+    // Method to get the user name of the user
+    public static String getUserName(String email) {
+        ClientIdentity identity = new ClientIdentity(ClientIdentity.Type.email, email);
+        Accounting accounting = DAO.getAccounting(identity);
+        String userId = identity.getUuid();
+        JSONObject accountingObj = accounting.getJSON();
+        String user_name = "";
```
Comment (Akshat-Jain) [inline src/ai/susi/server/api/cms/GetSkillFeedbackService.java]: Please use camelcase for variable naming. Thanks :-)

**Comment by akshatnitd** [comment]
@arundhati24 Yes, right. The users who do not have a user name set, we will be showing the anonymised email id. But, if the user has his/her comment on the Feedback section, the full email should be shown.

**Comment by akshatnitd** [comment]
@Orbiter Made the suggested changes :v:

**Pull request merged**